### DISCUSSION TITLE
How to remove outline on the <Editor /> element

### DISCUSSION THREAD

**Discussion by 2coolchampion** [ianstormtaylor/slate#5624] [Q&A]
The \<Editor\/\> element by has an outline whenever I click within it.
In my case the outline is white and I don't want it.

There is absolutely nothing I can find related to what is setting that outline in the styles tab, computed tab or even the css file in the sources.

Here is what it looks like:

![vivaldi_zW1AdYEJei](https://github.com/ianstormtaylor/slate/assets/105818200/d2954ed2-95a2-4dc2-abb2-ea88f46ff21b)

How can I remove it?

**Comment by Casewinter** [comment]
If I’m not wrong, pass classname with "outline: none" should work

**Reply by 2coolchampion** [reply]
Well that's embarrassing... Thank you.
